### PR TITLE
Proposed changes/updates to wicket-requirejs

### PR DESCRIPTION
**Pull request by christofer-f**
Modification of both baseUrl and mountPath to support deployment to application root other than / (i.e. Wildfly, Tomcat etc)

Updated references to wicket 7

### CODE REVIEW

**Comment by l0rdn1kk0n** [comment]
thanks.

File: pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
     <groupId>de.agilecoders.wicket.requirejs</groupId>
     <artifactId>wicket-requirejs</artifactId>
-    <version>0.1.3-SNAPSHOT</version>
+    <version>0.1.3-SNAPSHOT_UPDATES</version>
```
Comment (martin-g) [inline pom.xml:13]: 0.2.0-SNAPSHOT?

File: pom.xml [hunk 2]
```diff
@@ -36,11 +36,11 @@
 
         <rhino.version>1.7.6</rhino.version>
         <closure.version>v20131014</closure.version>
-        <wicket.version>6.19.0</wicket.version>
+        <wicket.version>7.0.0</wicket.version>
```
Comment (martin-g) [inline pom.xml:39]: 7.1.0?

File: src/test/java/de/agilecoders/wicket/requirejs/RequireJsConfigHeaderItemTest.java [hunk 3]
```diff
@@ -26,26 +26,26 @@ public void correctMapIsRendered() throws Exception {
...
+//                                              + "      \"deps\": [\"jquery\"]\n"
+//                                              + "    },\n"
+//                                              + "    \"Wicket\": {\n"
+//                                              + "      \"exports\": \"Wicket\",\n"
+//                                              + "      \"deps\": [\"wicket-event\"]\n"
+//                                              + "    }\n"
+//                                              + "  },\n"
+//                                              + "  \"waitSeconds\": 600,\n"
+//                                              + "  \"mountPath\": \"../wicket/requirejs\"\n"
+//                                              + "});"));
```
Comment (martin-g) [inline src/test/java/de/agilecoders/wicket/requirejs/RequireJsConfigHeaderItemTest.java:48]: Should this be fixed or removed?

**Pull request merged**